components/patient-access-rules/ui/src/main/resources/PhenoTips/PatientAccessRightsManagement.xml

```diff
@@ -94,12 +94,36 @@
   #set ($collaborators = $request.getParameterValues('collaborator'))
   #set ($accessLevels = $request.getParameterValues('accessLevel'))
   #set ($owner = "$!{request.get('owner')}")
+  #if ("$!{owner}" != "")
+    ##check the validity of the new owner
+    #set($ownerDoc = $services.model.resolveDocument($owner))
+    #set($ownerDocRef = $services.model.resolveDocument("$owner"))
+    #if ($xwiki.getDocument($ownerDocRef).getObject("PhenoTips.PhenoTipsGroupClass"))
+      #set($validGroup = true)
+      ##check to see if user has permission to transfer ownership
```

Comment:
This check isn't valid. Any group member can transfer ownership **to** the group, but only group admin can transfer ownership **away from** it.